### DISCUSSION TITLE
Suggestion database wrongly reports `reexported_in` for Constructor entries.

### DISCUSSION THREAD

**Discussion by vitvakatu** [enso-org/enso#7671] [Work In Progress - Engine]
## Issue
While working on #7468, I was asked to insert `Boolean.True` and `Boolean.False` types in dropdowns without unnecessary `Boolean` prefixes. After further investigation, it proved to be impossible with the current suggestion database.

Technically, we should be able to see that `Boolean.True` constructor is reexported by the `Standard.Base` module and, as we have `from Standard.Base import all`, neither importing nor fully qualified name is required. But the suggestion database has a `reexported_in` field set for the Type and all its Constructors. So we can't distinguish whether the _Constructor_ was reexported or only its type was.

## Example 

1. `Boolean.True` is reexported to the `Standard.Base`. It is shown in the `reexported_in` field:
```json
{
  "name": {
    "content": "True"
  },
  "in": "Standard.Base.Data.Boolean",
  "reexported_in": "Standard.Base"
}
```
2. A lot of other constructors are not reexported directly, though. One of the examples is `Standard.Base.Data.Regression.Model.Logarithmic`:
```json
{
  "name": {
    "content": "Logarithmic"
  },
  "in": "Standard.Base.Data.Regression",
  "reexported_in": "Standard.Base"
}
```

As you can see, we can't distinguish between `Boolean.True` and `Logarithmic`, and therefore can't use `True` without the `Boolean` prefix.

**Comment by 4e6** [comment]
You're right, `Logarithmic` is not re-exported from `Standard.Base`. There is an issue with an algorithm that sets the re-export field, I'll check it.

**Reply by 4e6** [reply]
Tracked by the enso-org/enso#7674
